### PR TITLE
DBVG-2

### PR DESCRIPTION
**Pull request by Grizzielicious**
Se agregan DAO, Service y Controllers para plataforma

### CODE REVIEW

File: src/main/java/org/grizzielicious/VideoGames/controllers/GeneroController.java [hunk 1]
```diff
@@ -3,15 +3,22 @@
...
 import org.grizzielicious.VideoGames.service.GeneroService;
+import org.grizzielicious.VideoGames.utils.ErrorUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.validation.Errors;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
+import java.util.Objects;
```
Comment (mralcaraz) [inline src/main/java/org/grizzielicious/VideoGames/controllers/GeneroController.java]: Remove unused imports

File: src/main/java/org/grizzielicious/VideoGames/dao/PlataformaDao.java [hunk 2]
```diff
@@ -2,18 +2,26 @@
...
 
+@Repository
 public interface PlataformaDao extends JpaRepository<Plataforma, Integer> {
 
     Optional<Plataforma> findByIdPlataforma(@Param("idPlataforma") int idPlataforma);
 
     Optional<Plataforma> findByPlataforma(@Param("plataforma") String Plataforma);
 
-    List<Plataforma> findByPlataformaLike(String Plataforma);
+    List<Plataforma> findByPlataformaContains(String Plataforma); //THE Jr has beaten up a Sr!!! Ü
```
Comment (mralcaraz) [inline src/main/java/org/grizzielicious/VideoGames/dao/PlataformaDao.java]: Remove useless and ridiculous comment.

File: src/main/java/org/grizzielicious/VideoGames/dto/Estudio.java [hunk 3]
```diff
@@ -20,7 +20,8 @@
 @Table(name="estudio")
 public class Estudio implements Serializable {
 
-    @Serial
+
+    @Serial //TODO preguntar si es necesario dado que marca warn si no se tiene
```
Comment (mralcaraz) [inline src/main/java/org/grizzielicious/VideoGames/dto/Estudio.java]: Remove comments

File: src/main/java/org/grizzielicious/VideoGames/exceptions/handlers/GenericControllerExceptionHandler.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import org.grizzielicious.VideoGames.exceptions.*;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+
+@ControllerAdvice(assignableTypes = {GeneroController.class, PlataformaController.class})
+@Order(Ordered.HIGHEST_PRECEDENCE) //TODO investigar
```
Comment (mralcaraz) [inline src/main/java/org/grizzielicious/VideoGames/exceptions/handlers/GenericControllerExceptionHandler.java]: Remove comment

**Review by mralcaraz — CHANGES_REQUESTED** [review]

File: src/main/java/org/grizzielicious/VideoGames/controllers/GeneroController.java (hunk 1 above)
Comment (Grizzielicious) [reply src/main/java/org/grizzielicious/VideoGames/controllers/GeneroController.java]: Done

File: src/main/java/org/grizzielicious/VideoGames/dao/PlataformaDao.java (hunk 2 above)
Comment (Grizzielicious) [reply src/main/java/org/grizzielicious/VideoGames/dao/PlataformaDao.java]: Done...

**Review by mralcaraz — APPROVED** [review]
Fix your tone, boy

**Pull request merged**